lib/bundler/cli/common.rb

```diff
@@ -98,5 +98,25 @@ def self.clean_after_install?
       clean &&= !Bundler.use_system_gems?
       clean
     end
+
+    def self.set_path(path, if_given = false)
+      if path.nil?
+        Bundler.settings.set_command_option :path, nil unless if_given
+        return
+      end
+
+      path = Pathname.new(path)
+      if path.relative? && Bundler.feature_flag.path_relative_to_cwd?
+        path = path.expand_path
+        root = Bundler.root
```

Comment:
i don't think `root` needs to be in a variable here

Reply:
this avoids the `rescue` effecting the `Bundler.root` call